lib/bamboozled/api/base.rb

```diff
@@ -35,7 +35,11 @@ def request(method, path, options = {})
           case response.code
           when 200..201
             begin
-              JSON.parse(response).with_indifferent_access
+              if response.nil?
+                {"headers" => response.headers}.with_indifferent_access
```

Comment:
@kylefdoherty the problem is that `response.nil?` will never be true  in this case so that first condition will never execute. You are right, `response.body.nil?` will not work either. `response.body.empty?` makes more sense. You can also do `response.body.to_s.empty?` if you want to be extra safe :smile: 

Can you share a gist of the fixture you are trying to use? I would think that you only need to add the headers with no data after them and it should be parsed correctly.